packages/login/theme/lumo/vaadin-login-form-styles.js

```diff
@@ -4,7 +4,7 @@ import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themab
 registerStyles(
   'vaadin-login-form',
   css`
-    vaadin-button[part='vaadin-login-submit'] {
```

Comment:
As `vaadin-login-form` doesn't have a shadow root, this rule would affect all buttons.

`vaadin-login-form-wrapper > form > vaadin-button` should work, although I feel that having some additional selector (class or attribute) would be safer, for example if we implement a feature that allows slotting content into the form.

Reply:
Sorry, meant to point at the line below: https://github.com/vaadin/web-components/pull/4889/files#diff-1e4cdd3af162f721f82da27558637c31684b48ea60a98409256ca8c18c28116eR6

Reply:
Good point, fixed by changing `theme` to include `submit` and using it to apply styles in themes.